com.unity.netcode.gameobjects/Runtime/Metrics/INetworkMetrics.cs

```diff
@@ -36,11 +36,11 @@ internal interface INetworkMetrics
 
         void TrackObjectDestroyReceived(ulong senderClientId, ulong networkObjectId, string gameObjectName, long bytesCount);
 
-        void TrackRpcSent(ulong receiverClientId, ulong networkObjectId, string rpcName, long bytesCount);
+        void TrackRpcSent(ulong receiverClientId, ulong networkObjectId, string rpcName, string networkBehaviourName, long bytesCount);
 
-        void TrackRpcSent(ulong[] receiverClientIds, ulong networkObjectId, string rpcName, long bytesCount);
+        void TrackRpcSent(ulong[] receiverClientIds, ulong networkObjectId, string rpcName, string networkBehaviourName, long bytesCount);
 
-        void TrackRpcReceived(ulong senderClientId, ulong networkObjectId, string rpcName, long bytesCount);
+        void TrackRpcReceived(ulong senderClientId, ulong networkObjectId, string rpcName, string networkBehaviourName, long bytesCount);
```

Comment:
nit, even these are a bit wide a ~140, but a bit more of a judgment call than the others